Library/Homebrew/livecheck/livecheck.rb

```diff
@@ -274,6 +314,22 @@ def checkable_urls(formula)
       urls.compact
     end
 
+    def checkable_cask_urls(cask)
+      urls = []
+      urls << cask.appcast.to_s if cask.appcast
+      urls << cask.url.to_s
+      urls << cask.homepage if cask.homepage
+      urls.compact
+    end
+
+    def checkable_urls(formula_or_cask)
```

Comment:
Since `#checkable_cask_urls` and `#checkable_formula_urls` are only used within `#checkable_urls` and all the `#checkable_urls` method contains is an if/else block that calls the specific functions, why don't we just incorporate this in one `#checkable_urls` method?

This is what I'm thinking:

```ruby
def checkable_urls(formula_or_cask)
  urls = []

  case formula_or_cask
  when Formula
    urls << formula.head.url if formula.head
    if formula.stable
      urls << formula.stable.url
      urls.concat(formula.stable.mirrors)
    end
    urls << formula.homepage if formula.homepage
  when Cask::Cask
    urls << cask.appcast.to_s if cask.appcast
    urls << cask.url.to_s if cask.url
    urls << cask.homepage if cask.homepage
  end

  urls.compact
end
```

Reply:
> why don't we just incorporate this in one `#checkable_urls` method?

Mostly to prevent Git from making the diff ugly, haha